resources/views/profile/update-profile-information-form.blade.php

```diff
@@ -1,7 +1,7 @@
 <div>
     <div class="flex flex-col">
-        <span class="text-2xl font-semibold text-theme-secondary-900">@lang('fortify::pages.user-settings.contact_information_title')</span>
-        <span>@lang('fortify::pages.user-settings.contact_information_description')</span>
+        <span class="header-4 mt-4">@lang('fortify::pages.user-settings.contact_information_title')</span>
```

Comment:
why `mt-4` here?

Reply:
Would look like https://gyazo.com/0631f1c7609f321a3ce57e6005864261 if not, while design show https://gyazo.com/5edf96ce25c5f0edd488d2ec0adaa831